en/04_Changelogs/5.0.0.md

```diff
@@ -26,9 +34,65 @@ title: 5.0.0 (unreleased)
   - [Other features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the [list Commercially Supported Modules for Silverstripe CMS 5](/Project_Governance/Supported_Modules).
+
+### Module not supported in Silverstripe CMS 5
+Some modules that were supported in Silverstripe CMS 4 will not be supported in Silverstripe CMS 5. 
+
+Just because a module is not supported officially, doesn't mean that you shouldn't be using it. However, it does mean that it's more likely to suffer from regressions. You should take this in consideration when choosing to install an unsupported module in your project.
```

Comment:
```suggestion
Just because a module is not supported officially, doesn't mean that you shouldn't be using it. Unsupported modules are maintained on a "best-effort" basis. You should take this into consideration when choosing to install an unsupported module in your project.
```

Reply:
I think I change something that "outdated" this comment. I'll bring it back.